rd-net/Test.RdFramework/Reflection/data/Generated/RefRoot.cs

```diff
@@ -62,7 +62,6 @@ public static void RegisterDeclaredTypesSerializers(ISerializers serializers)
     {
       
       serializers.RegisterToplevelOnce(typeof(RefRoot), RefRoot.RegisterDeclaredTypesSerializers);
-      serializers.RegisterToplevelOnce(typeof(RefExt), RefExt.RegisterDeclaredTypesSerializers);
```

Comment:
This change shouldn't be applied. It is, most likely, caused by a class loading bug in rdgen.